libs/ui-lib/lib/common/components/clusterWizard/types.ts

```diff
@@ -33,6 +33,7 @@ export type ClusterDetailsValues = {
   cpuArchitecture: string;
   platform: PlatformType;
   customOpenshiftSelect: OpenshiftVersionOptionType | null;
+  helperTextOpenshift: JSX.Element | null;
```

Comment:
we should not store the JSX.Element in state. Just calculate it when needed.

Reply:
I'll try it but not works because versions are different inside the modal and in the cluster wizard